css-will-change-transform-rasterization/index.html

```diff
@@ -0,0 +1,58 @@
+---
+feature_name: "Rasterization &amp; will-change: transform"
+chrome_version: 53
+feature_id: 5637351992721408
+---
+
+<h3>Background</h3>
+<p>
+  All content that does not have the <code>will-change: transform</code> CSS
+  property will be re-rastered when its transform scale changes. In other words,
+  <code>will-change: transform</code> effectively means "please animate it fast"
```

Comment:
"please animate it fast" is a bit confusing because the demo doesn't actually show an animation. It shows a transform with this before/after behavior.